BunnymodXT/modules/HwDLL.cpp

```diff
@@ -5586,24 +5586,22 @@ HOOK_DEF_1(HwDLL, int, __cdecl, Host_FilterTime, float, passedTime)
 
 	auto &hw = HwDLL::GetInstance();
 
-	static bool is_0x75 = false;
-
+	static bool is_0x7E_on_init = false;
 	if (pHost_FilterTime_FPS_Cap_Byte)
 	{
-		if (CVars::bxt_remove_fps_limit.GetBool())
-		{
-			if (*reinterpret_cast<byte*>(pHost_FilterTime_FPS_Cap_Byte) == 0x75)
-				is_0x75 = true;
+		const auto pByte = *reinterpret_cast<byte*>(pHost_FilterTime_FPS_Cap_Byte);
+		if (pByte == 0x7E)
+			is_0x7E_on_init = true;
```

Comment:
```suggestion
		static bool is_0x7E_on_init = (pByte == 0x7E);
```
and remove the declaration above